delta/plugins/elasticsearch/src/main/resources/defaults/default-mapping.json

```diff
@@ -182,6 +182,9 @@
         }
       }
     },
+    "_keywords": {
```

Comment:
keywords.
Also:
* You will need a multi-field here to enable full text search properly
* `Flattened` could be an option here, we have to be careful that when searching, the ES query does not fail because a keyword value does not exist but return an empty value